### PR TITLE
Generic webhook - integration cache improvements

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/40193

## Description
- implemented the protection of storing events of size less than size 1MB in the integration context (as done in cs streaming v2)
- added the option to clear the integration context

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Documentation 

Note: not bumping docker image tag cause of https://github.com/tiangolo/fastapi/issues/3373

### CODE REVIEW

File: Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.py [hunk 1]
```diff
@@ -87,12 +87,16 @@ async def handle_post(
 
     if demisto.params().get('store_samples'):
         try:
-            sample_events_to_store.append(incident)
-            integration_context = get_integration_context()
-            sample_events = deque(json.loads(integration_context.get('sample_events', '[]')), maxlen=20)
-            sample_events += sample_events_to_store
-            integration_context['sample_events'] = list(sample_events)
-            set_to_integration_context_with_retries(integration_context)
+            incident_obj_size = sys.getsizeof(incident)
```
Comment (glicht) [inline Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.py:90]: I think this doesn't do a recursive calculation. Please check.

Comment (Itay4) [reply Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.py]: right
implemented an `actualsize` func

**Review by yaakovi — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@glicht @yaakovi @anara123 Doc review completed.

**Comment by yaakovi** [comment]
@Itay4 what is the status here?

**Pull request closed without merging**